test/unit-test/FreeRTOS_ARP/FreeRTOS_ARP_utest.c

```diff
@@ -32,29 +32,32 @@ extern NetworkBufferDescriptor_t * pxARPWaitingNetworkBuffer;
 
 extern BaseType_t xARPHadIPClash;
 
+uint8_t ucARPPacket[ ipconfigETHERNET_MINIMUM_PACKET_BYTES ];
+uint8_t ucARPPacket2[ ipconfigETHERNET_MINIMUM_PACKET_BYTES ];
```

Comment:
Why is ucARPPacket2 is required?  Is ucARPPacket not enough? I can see only first test case is using it.

Reply:
`ucARPPacket2 ` is not used by test case.  `vResetARPClashCounter` is used to reset `uxARPClashCounter` while we are running a test case, so it requires a different memory other than the one used for running the tests.

Reply:
Oh can we move line 38 to  71 then, i was confused as it mentioned test case.